ArcadiaParties-SPA/src/app/routes.ts

```diff
@@ -0,0 +1,11 @@
+import { Routes } from '@angular/router';
+import { AppComponent } from './app.component';
+
+const routes: Routes = [
```

Comment:
Why do you need this?

Reply:
Already deleted.
Also added children routes.